lib/Runtime/PlatformAgnostic/Platform/Windows/NumbersUtility.cpp

```diff
@@ -0,0 +1,34 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "RuntimePlatformAgnosticPch.h"
+#include "Common.h"
+#include "ChakraPlatform.h"
+
+namespace PlatformAgnostic
+{
+namespace Numbers
+{
+    size_t Utility::NumberToDefaultLocaleString(const WCHAR *number_string,
+                                                const size_t length,
+                                                WCHAR *buffer,
+                                                const size_t pre_allocated_buffer_size)
+    {
+        size_t count = GetNumberFormatEx(LOCALE_NAME_USER_DEFAULT, 0,
```

Comment:
GetNumberFormatEx returns an int, not a size_t. I'd change this to size_t and assert for count >= 0 before using it as a size_t. I dont know why we were checking for <= 0 before your change since the docs say its either 0 or greater.